src/librustc_mir/build/block.rs

```diff
@@ -91,8 +91,14 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
         let source_info = this.source_info(span);
         for stmt in stmts {
             let Stmt { kind, opt_destruction_scope } = this.hir.mirror(stmt);
+            let kind2 = kind.clone();
```

Comment:
oops this binding of `kind2` (and associated clone) wasn't supposed to be part of the PR. Will fix.